src/dbcore/dbcore-indexeddb.ts

```diff
@@ -39,12 +39,9 @@ export function getKeyPathAlias(keyPath: null | string | string[]) {
 
 export function createDBCore (
   db: IDBDatabase,
-  indexedDB: IDBFactory,
   IdbKeyRange: typeof IDBKeyRange,
   tmpTrans: IDBTransaction) : DBCore
 {
-  const cmp = indexedDB.cmp.bind(indexedDB);
```

Comment:
Hello, i was very excited when i first read about those performance improvements, however when taking a look at the code i got some doubts.

- Have to tried to perform those benchmarks without the use of `Function.bind` (?)
- This JS version of cmp seems only faster for small amounts of data, and specially when the data isn't equal from the first few bytes (?)

Check this out:

```js
b1=cryptHelper.getRandomValues(473)
b2=cryptHelper.getRandomValues(473)

a1 = () => indexedDB.cmp(b1,b2)
a2 = () => compareUint8Arrays(b1,b2)
```

```js
> console.time('dd');for(let i = 1e6; i--; a1());console.timeEnd('dd');
< dd: 799.961181640625 ms

> console.time('dd');for(let i = 1e6; i--; a2());console.timeEnd('dd');
< dd: 5.804931640625 ms
```

All good here, with 473bytes of data and not equal.

```js
b2=b1
> console.time('dd');for(let i = 1e6; i--; a1());console.timeEnd('dd');
< dd: 822.533203125 ms

> console.time('dd');for(let i = 1e6; i--; a2());console.timeEnd('dd');
< dd: 702.003173828125 ms
```

Still reasonable here when the data is equal, but...

```js
b1=cryptHelper.getRandomValues(4730)
b2=b1

> console.time('dd');for(let i = 1e6; i--; a1());console.timeEnd('dd');
< dd: 1101.98681640625 ms

> console.time('dd');for(let i = 1e6; i--; a2());console.timeEnd('dd');
< dd: 8124.588134765625 ms
```

Thoughts? Not sure if i did overlooked something, those tests were made under Chrome 96 on Windows.